### DISCUSSION TITLE
Tiled for Android

### DISCUSSION THREAD

**Discussion by junimobrat** [mapeditor/tiled#3056] [General]
Hello, sir. Will the Tiled mapeditor be available for android phones?

**Comment by bjorn** [comment]
Tiled as currently implemented is a desktop application that would not be suitable for publishing to Android phones. A version for Android would mean a complete rewrite of the entire UI to make it fit on smaller screens and usable with fingers. I have at some point made a start on a new application based on Qt Quick, which is currently functional only as a map viewer that also works on Android (in the `src/tiledquick` folder in the repository). But I'm not sure if I'll ever have time to turn this into a full editor.

If you want to edit maps on your Android device now, I'd recommend checking out [NotTiled](https://play.google.com/store/apps/details?id=com.mirwanda.nottiled) by @wandsmire. It's not Tiled, but it's generally compatible since it works with the same map format.

See also issue #1971.
